back/src/companies/resolvers/queries/companyExhaustive.ts

```diff
@@ -0,0 +1,124 @@
+import { QueryResolvers } from "../../../generated/graphql/types";
+
+import { checkIsAdmin } from "../../../common/permissions";
+import { applyAuthStrategies, AuthType } from "../../../auth";
+import { prisma } from "@td/prisma";
+import { searchCompanyTD } from "../../sirene/trackdechets/client";
+import { searchCompanySirene } from "../../sirene/insee/client";
+import { AnonymousCompany, Company } from "@prisma/client";
+import { SireneSearchResult } from "../../sirene/types";
+import { isSiret } from "@td/constants";
+import { UserInputError } from "../../../common/errors";
+
+const formatEtatAdministratif = etat => {
+  if (etat === "A") return "A (Active)";
+  if (etat === "F") return "F (Fermé)";
+  if (etat === "C") return "C (Cessée)";
```

Comment:
Pour qu'on voit tout de suite la diff on pourrait p-e indiquer le suivant (si j'ai bien compris ce qu'on a vu tout à l'heure):
```suggestion
  if (etat === "F") return "F (Etablissement fermé)";
  if (etat === "C") return "C (Unité légale cessée)";
```